core/src/main/java/hudson/security/SidACL.java

```diff
@@ -23,16 +23,15 @@
  */
 package hudson.security;
 
-import org.acegisecurity.Authentication;
-import org.acegisecurity.GrantedAuthority;
-import org.acegisecurity.acls.sid.PrincipalSid;
-import org.acegisecurity.acls.sid.GrantedAuthoritySid;
-import org.acegisecurity.acls.sid.Sid;
-
 import edu.umd.cs.findbugs.annotations.NonNull;
-import java.util.logging.Logger;
 import static java.util.logging.Level.FINE;
 import static java.util.logging.Level.FINER;
+import java.util.logging.Logger;
+import org.acegisecurity.acls.sid.GrantedAuthoritySid;
+import org.acegisecurity.acls.sid.PrincipalSid;
+import org.acegisecurity.acls.sid.Sid;
```

Comment:
No particular reason to switch to the Spring Security version of this package, which would require pulling in the `spring-security-acl` module as well as disruption to existing `AuthorizationStrategy`s: so long as you stick to the `String`-based constructors, there is no conflict with usage of other Spring Security types.

Reply:
What do you mean? It is not included as a dependency by this PR.

Reply:
I meant as a precautionary measure so that downstream plugins don't think they should switch to that library just because of the rest of the Spring Security upgrade. I don't know if there's anything worth preventing, though, beyond confusion.